wdl-analysis/src/analyzer.rs

```diff
@@ -64,6 +70,11 @@ pub enum ParseResult {
     Error(Arc<anyhow::Error>),
     /// The document was parsed.
     Parsed {
+        /// The version of the parsed document.
```

Comment:
I might clarify this is the monotonic version (just to delineate from the WDL version).

Reply:
I see you resolved this, but I didn't see any update to the comment. What's your
strategy on that @peterhuene?

Reply:
I accidentally pushed up a clarification on the internal representation of this rather than the public API.

I'll fix!